pkgs/development/python-modules/readability-lxml/default.nix

```
@@ -31,8 +31,12 @@ buildPythonPackage rec {
     timeout-decorator
   ];
 
+  disabledTests = [
```

```suggestion
  disabledTests = lib.optionals stdenv.isDarwin [
```
Sorry to bother you again, but this is only needed for darwin. Could you please add a conditional branch?